app/modules/questionnaire/server.R

```diff
@@ -1,7 +1,6 @@
-source('app/modules/questionnaire/generate_questions_UI.R')
-source('app/modules/questionnaire/generate_recommendation.R')
-
-questions_df <- read.csv("app/data/questionnaire_inputs/q_database.csv")
+here::i_am("app/modules/questionnaire/server.R")
+data_directory <- here('app','data','questionnaire_inputs') #define relative path to your questionnaire app directory
+questions_df <- read.csv(here(data_directory,"q_database.csv"))
```

Comment:
I found the issue - here is not called explicitly for this two line.

```suggestion
data_directory <- here::here('app','data','questionnaire_inputs') #define relative path to your questionnaire app directory
questions_df <- read.csv(here::here(data_directory,"q_database.csv"))
```